src/main/java/org/testng/internal/Utils.java

```diff
@@ -167,8 +166,8 @@ private static void writeFile(@Nullable File outDir, String fileName, String sb,
       File outputFile = new File(outDir, fileName);
       if (!append) {
         outputFile.delete();
-        log("Attempting to create " + outputFile);
-        log("  Directory " + outDir + " exists: " + outDir.exists());
+        log("[Utils]", 3, "Attempting to create " + outputFile);
```

Comment:
set log level to 3 to be consistent with the existing logging: https://github.com/cbeust/testng/blob/master/src/main/java/org/testng/internal/Utils.java#L192